### PR TITLE
multiprocess support for task manager

### PR DESCRIPTION
**Pull request by solarw**
## Proposed changes

multiprocess support for task manager

## Fixes

https://github.com/fetchai/agents-aea/issues/2259

## Types of changes

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

- [ ] I have read the [CONTRIBUTING](../CONTRIBUTING.md) doc
- [ ] I am making a pull request against the `develop` branch (left side). Also you should start your branch off our `develop`.
- [ ] Lint and unit tests pass locally with my changes and CI passes too
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that code coverage does not decrease.
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

DELETE INCLUSIVE THIS AND BELOW FOR STANDARD PR
------

## Release summary

Version number: [e.g. 1.0.1]

## Release details

Describe in short the main changes with the new release.

### CODE REVIEW

File: aea/skills/tasks.py [hunk 1]
```diff
@@ -98,33 +107,69 @@ def execute(self, *args: Any, **kwargs: Any) -> Any:
     def teardown(self) -> None:
         """Implement the task teardown."""
 
-
-def init_worker() -> None:
+def _init_worker(mode: str, packages: Dict[str, List[Dict[str, str]]]) -> None:
```
[the remark this answers is not in the document]
Comment (solarw) [reply aea/skills/tasks.py:110]: actual;ly it's internal method that shoud not be called directly. not a public api, this is why i renamed it with underscore

File: packages/fetchai/contracts/fet_erc20/contract.yaml [hunk 2]
```diff
@@ -8,6 +8,7 @@ aea_version: '>=1.0.0, <2.0.0'
 fingerprint:
   README.md: QmdksAfBj3gVPH8zmzpHjjpXsf1gtAY6z6gjrfYfiicLrr
   __init__.py: QmeJ8x6NiZjVLFbUTgpPPCMzQscqjHNBDExapiezpakksR
+  bash: QmcPNXE5zjkWkM24xQ7Bi3VAm8fRxiaNp88jFsij7kSQF1
```
[the remark this answers is not in the document]
Comment (solarw) [reply packages/fetchai/contracts/fet_erc20/contract.yaml:11]: done

File: aea/components/base.py [hunk 3]
```diff
@@ -116,24 +116,42 @@ def build_directory(self) -> Optional[str]:
...
+    Load the AEA package from configuration.
 
     It adds all the __init__.py modules into `sys.modules`.
 
     :param configuration: the configuration object.
     """
     dir_ = configuration.directory
     if dir_ is None:  # pragma: nocover
-        raise AEAEnforceError("configuration directory does not exists.")
+        raise AEAEnforceError(f"configuration directory `{dir_}` does not exists.")
```
[the remark this answers is not in the document]
Comment (solarw) [reply aea/components/base.py:127]: done

**Review by DavidMinarsch — APPROVED** [review]
LGTM

**Review by 5A11 — APPROVED** [review]

**Pull request merged**